homeassistant/components/unifi/sensor.py

```diff
@@ -138,6 +139,12 @@ class UnifiSensorEntityDescriptionMixin(Generic[HandlerT, ApiItemT]):
     value_fn: Callable[[UniFiController, ApiItemT], datetime | float | str | None]
 
 
+@callback
+def async_device_state_value_fn(controller: UniFiController, device: Device) -> str:
+    """Retrieve the state of the device."""
+    return DEVICE_STATES[device.state]
```

Comment:
Just in case unifi might introduce new values, wouldn't it be better to default to UNKNOWN in case we don't find the state?

Reply:
The enum should default to unknown by itself.